oneapp-server/types/ApplicationProgramInfo.js

```diff
@@ -0,0 +1,52 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Application {
+    programInfo: ApplicationProgramInfo
+  }
+
+  type ApplicationProgramInfo {
```

Comment:
Opted to not include the `MA_` fields found in the database, as they pertain to Medicaid, which is no longer supported in OneApp.

Also opted to not include the `IS_*_ELIGIBLE` and `*_ELIGIBLE_AMOUNT` fields as those will be calculated via a different stored procedure / GraphQL type. Thinking we will still persist the fields in the database, so as I'm typing this, maybe we do want to include it so the front-end can access the version stored in the db and doesn't have to recalculate each time?